projects/mysadna/src/app/components/contacts/contacts.component.scss

```diff
@@ -14,6 +14,9 @@
     padding-left: $margin;
     padding-top: $margin;
     text-align: start;
+    @media(max-width: 579px) {
```

Comment:
Since we're discussing implementation right now in slack, let's remove the change for awhile.

Reply:
Alright.